pkg/kdump/kdump.sh

```diff
@@ -25,11 +25,18 @@ if test -f /proc/vmcore; then
     while read -r line; do echo "$line" > /dev/kmsg; done < /tmp/backtrace
     echo ">>>>>>>>>> Crashed kernel dmesg END <<<<<<<<<<" > /dev/kmsg
 
+    TS=$(date +%Y-%m-%d-%H-%M-%S)
```

Comment:
Why not use date -Is instead? That as the same info but in the standard ISO format.

Reply:
No any reason. Will update pr.